saleor/product/constants.py

```diff
@@ -0,0 +1,9 @@
+from ..permission.enums import DiscountPermissions, OrderPermissions, ProductPermissions
```

Comment:
Should this file be called `constants`? Or do we have another preferred naming?

Moving this out from `models` as it's not even used there and instead it's imported all around the place everywhere else.

Reply:
We tend to put such constants in `__init__` files of the modules where they are commonly used or belong by domain.